rules/python3.mk

```diff
@@ -0,0 +1,38 @@
+# python3.5.2+
```

Comment:
snmpagent depends on python3.5.2+. Currently 3.6.0-1 was the only backport-able version for Debian Jessie fulfill our requirement. So I keep the filename and the comment as accurate as possible, in order not to mislead that 3.6.0-1 is the lowest require version.

---
In reply to: [99676879](https://github.com/Azure/sonic-buildimage/pull/245#discussion_r99676879) [](ancestors = 99676879)